From 63a7311d95743e14b4cb152132f20795c117585f Mon Sep 17 00:00:00 2001
From: chen-dq <chen.dq@neusoft.com>
Date: Thu, 12 Sep 2019 07:58:25 +0000
Subject: [PATCH] add_makefile

---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..cda58eb
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,2 @@
+build:
+	echo "no build"
-- 
2.23.0

