From f94503b355fdcb7ba31e5f95053ed2010ed6d997 Mon Sep 17 00:00:00 2001
From: fuyong <fuyong@neusoft.com>
Date: Wed, 18 Sep 2019 16:56:17 +0800
Subject: [PATCH] fix Makefile dir

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 95a8300..8a5a514 100644
--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@
 # Copyright (C) 2019 Intel Corporation
 #
 
-MODULEDIR ?= /usr/share/puppet/modules
+MODULEDIR ?= %{buildroot}/usr/share/puppet/modules
 
 install:
 	install -d -m 0755 $(MODULEDIR)/dcmanager
-- 
2.23.0

