From 33c6f75269f8d0e86d8401121b923ec72a8176d9 Mon Sep 17 00:00:00 2001
From: Mingyuan Qi <mingyuan.qi@intel.com>
Date: Tue, 11 Sep 2018 00:45:39 +0800
Subject: [PATCH 1/2] strncpy specified bound equals destination size

---
 sources/fmAlarmUtils.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sources/fmAlarmUtils.cpp b/sources/fmAlarmUtils.cpp
index 81c353b..41327a8 100644
--- a/sources/fmAlarmUtils.cpp
+++ b/sources/fmAlarmUtils.cpp
@@ -42,7 +42,8 @@ static stoi_t err_to_int;
 			((str).length()==1 && ((str).c_str())[0]==' ')) { \
 			memset(charb,0,sizeof(charb)); \
 		} else {  \
-			strncpy((charb),str.c_str(),sizeof(charb)); \
+			strncpy((charb),str.c_str(),sizeof(charb)-1); \
+			(charb)[sizeof(charb)-1] = '\0'; \
 		}
 
 void add_both_tables(int id, const char *str, itos_t &t1,stoi_t &t2 ) {
-- 
2.18.0

