From be932e33576769196cb6e5966bfed31e457b45c7 Mon Sep 17 00:00:00 2001
From: chen-dq <chen.dq@neusoft.com>
Date: Wed, 18 Sep 2019 01:41:50 +0000
Subject: [PATCH 1/3] lighttpd-1.4.39-defaultconf

---
 doc/config/lighttpd.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/config/lighttpd.conf b/doc/config/lighttpd.conf
index 7673535..fc1896a 100644
--- a/doc/config/lighttpd.conf
+++ b/doc/config/lighttpd.conf
@@ -14,7 +14,7 @@
 ## chroot example aswell.
 ##
 var.log_root    = "/var/log/lighttpd"
-var.server_root = "/srv/www"
+var.server_root = "/var/www"
 var.state_dir   = "/var/run"
 var.home_dir    = "/var/lib/lighttpd"
 var.conf_dir    = "/etc/lighttpd"
@@ -112,7 +112,7 @@ server.groupname = "lighttpd"
 ##
 ## Document root
 ##
-server.document-root = server_root + "/htdocs"
+server.document-root = server_root + "/lighttpd"
 
 ##
 ## The value for the "Server:" response field.
@@ -204,7 +204,7 @@ server.network-backend = "sendfile"
 ## By default lighttpd would not change the operation system default.
 ## But setting it to 2048 is a better default for busy servers.
 ##
-server.max-fds = 2048
+
 
 ##
 ## listen-backlog is the size of the listen() backlog queue requested when
-- 
2.23.0

