From c29aee9f13e2aae270d105d855abe02790918825 Mon Sep 17 00:00:00 2001
From: fuyong <fuyong@neusoft.com>
Date: Thu, 19 Sep 2019 14:28:55 +0800
Subject: [PATCH] fix Makefile dir

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d35b37f..8a8bc7c 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-DESTDIR ?= /usr/share/ansible/stx-ansible
+DESTDIR ?= %{buildroot}/usr/share/ansible/stx-ansible
 
 install:
 	install -d -m 0755 $(DESTDIR)/playbooks
-- 
2.23.0

