From 6fd3dfee1e189ca8f2732e1ac047e213447e4fb0 Mon Sep 17 00:00:00 2001
From: chen-dq <chen.dq@neusoft.com>
Date: Mon, 16 Sep 2019 06:40:40 +0000
Subject: [PATCH] add_makefile

---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..cda58eb
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,2 @@
+build:
+	echo "no build"
-- 
2.23.0

